sdk/monitor/Azure.Monitor.Query/tests/LogsQueryClientClientLiveTests.cs

```diff
@@ -675,6 +675,47 @@ public async Task CanQueryWithFormattedQuery(FormattableStringWrapper query)
             Assert.True(response.Value.Single());
         }
 
+        [RecordedTest]
+        public async Task CanQueryResource()
```

Comment:
The test recording is very large - is there a way to limit the size?

Reply:
removed recordings